app/models/message_generator.rb

```diff
@@ -5,6 +5,11 @@ def pairing_notification(name, date, time)
     "View it at #{paired_link}."
   end
 
+  def pairing_reminder(time)
+    "Friendly reminder that you have a pairing booked today at #{times[time]}. " +
```

Comment:
Can we change "today at" to "today from" for consistency with the other messages?

Reply:
Done.